src/controllers/user.controller.ts

```diff
@@ -91,7 +93,7 @@ export class UserController {
   }
 
   @post('/users/login')
-  async login(@requestBody() user: User): Promise<User> {
+  async login(@requestBody() user: User): Promise<string | undefined> {
```

Comment:
We need a type filter to describe the credentials needed for login.

The filtered type would be

```ts
type Credentials = {
   username: string,
  // consider adding `Password` and `Email` as built-in loopback types
   password: Password,
   email: Email
}
```

Or 

```ts
type Credentials = {
  identity: string | Email
  password: Password
}
```